desktop/apps/consign/client/actions.js

```diff
@@ -198,6 +198,16 @@ export function createSubmission () {
                            .set('Authorization', `Bearer ${token}`)
                            .send(inputs)
       }
+      let userBody
+      // update the user's phone number if it has been changed
+      if (user.phone !== inputs.phone) {
+        userBody = await request
+                           .put(`${sd.API_URL}/api/v1/me`)
+                           .set('X-ACCESS-TOKEN', user.accessToken)
+                           .send({ phone: inputs.phone })
+        dispatch(updateUserPhone(userBody.body.phone))
```

Comment:
We want to update the phone number on the `user` but not the entire `user` object (since the response we get back from this endpoint is slightly different from what we store on `sd.CURRENT_USER`).